### ISSUE TITLE
Warnings about path size and portability

### ISSUE THREAD

**Issue by bstaton1** [bstaton1/KuskoHarvEst#189]
When using `install_github("bstaton1/KuskoHarvEst")`, the user is greeted with a huge amount of warnings, all of the same variety. Here is an example:

```
 Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
     storing paths of more than 100 bytes is not portable:
     'KuskoHarvEst/inst/rstudio/templates/project/resources/05-example-data/CBM_2020_06_12_EXAMPLE_ONLY.csv'
   Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
     storing paths of more than 100 bytes is not portable:
     'KuskoHarvEst/inst/rstudio/templates/project/resources/05-example-data/Flight_counts_2020_06_12_EXAMPLE_ONLY.csv'
   Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
     storing paths of more than 100 bytes is not portable:
     'KuskoHarvEst/inst/rstudio/templates/project/resources/07-final-report-content/00-specify-odd-openers.R'
   Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
     storing paths of more than 100 bytes is not portable:
     'KuskoHarvEst/inst/rstudio/templates/project/resources/07-final-report-content/01-compile-all-estimates.R'
```

R is mad about my long file paths/names. So far this hasn't created an issue (hence low priority), but it sure is ugly and may lead to questions down the road.

I should consider shortening the file paths. This would be a pretty large change and require a huge amount of testing, as this is all about package internals. Further, I don't think I can shorten any of:

```
KuskoHarvEst/inst/rstudio/templates/project/resources
```

So this could get tricky.

- [x] Figure out how to determine the byte size of a file path
- [ ] Test whether I can put these files in the `rstudio` or `templates` directory to trim off - this would be the easiest route because it needs only a large cut/paste and changing the default behavior of `resource_path()`
- [ ] If needed, devise new names for subdirectories of the `resources` directory, e.g., `01-common`, `07-final-report-content` could be shortened
- [ ] Implement: change names
- [ ] Implement: change code calling the names

<details>
  <summary>Click to show all warnings</summary>

```
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/02-estimate-report/01a-data-sources_driftset.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/02-estimate-report/01b-data-sources_driftset_noset.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/02-estimate-report/01c-data-sources_setonly.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/02-estimate-report/02a-effort_driftset_1flight.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/02-estimate-report/02a-effort_driftset_2flight.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/02-estimate-report/02a-effort_driftset_3flight.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/02-estimate-report/02b-effort_driftset_noset_1flight.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/02-estimate-report/02b-effort_driftset_noset_2flight.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/02-estimate-report/02b-effort_driftset_noset_3flight.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/02-estimate-report/03b-harvest_driftset_noset.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/02-estimate-report/09b-nonsalmon_driftset_noset.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/03-sensitivity-report/01a-effort_1flight_noplots.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/03-sensitivity-report/01b-effort_1flight_plots.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/03-sensitivity-report/01c-effort_multiflight_noplots.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/03-sensitivity-report/01d-effort_multiflight_plots.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/03-sensitivity-report/02b-harvest_driftset_noset.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/03-interview-flight-data-tool.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/03-interview-flight-data-tool.html'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/00-project-summary.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/01-project-deliverables.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/02-workflow-description.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/03-what-is-rmarkdown.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/04-issues-solutions.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/05-versioning-hosting.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/06-future-directions.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/07-acknowldegements.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/08-references.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/09-appendix-a.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/10-appendix-b.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/2-fig/'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/2-fig/KuskoHarvEst-Workflow.drawio'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/2-fig/KuskoHarvEst-Workflow.pdf'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/2-fig/ex-report-1.pdf'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/2-fig/ex-report-2.pdf'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/2-fig/ex-report-3.pdf'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/2-fig/ex-report-4.pdf'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/2-fig/ex-sensitivity-1.pdf'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/2-fig/ex-sensitivity-2.pdf'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/2-fig/ex-sensitivity-3.pdf'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/3-submit/'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/3-submit/KuskoHarvEst-final-report.pdf'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/KuskoHarvEst-final-report.Rproj'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/_bookdown.yml'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/_output.yml'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/cites.bib'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/KuskoHarvEst-final-report/index.Rmd'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/Staton - 2018 - In-season harvest and effort estimates.pdf'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/00-general/active-project.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/00-general/new-KuskoHarvEst-project.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/00-general/new-directory.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/00-general/new-project-info.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/00-general/new-project.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/01-raw-data/bbh-data.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/01-raw-data/cbm-data.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/01-raw-data/fc-data.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/01-raw-data/flight-data.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/02-meta-data-tool/'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/02-meta-data-tool/announcement-info.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/02-meta-data-tool/calendar.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/02-meta-data-tool/contact.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/02-meta-data-tool/date-time.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/02-meta-data-tool/done.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/02-meta-data-tool/save.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/02-meta-data-tool/set-only.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/02-meta-data-tool/spatial.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/02-meta-data-tool/whole-tool.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/03-interview-flight-data-tool/'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/03-interview-flight-data-tool/done.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/03-interview-flight-data-tool/flight-plot.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/03-interview-flight-data-tool/flight-table.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/03-interview-flight-data-tool/global-options.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/03-interview-flight-data-tool/interview-buttons.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/03-interview-flight-data-tool/interview-output.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/03-interview-flight-data-tool/navbar-flights.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/03-interview-flight-data-tool/navbar-interview-options.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/03-interview-flight-data-tool/navbar-interview-output.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/03-interview-flight-data-tool/sources-to-include.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/03-interview-flight-data-tool/strata-gear-summary.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/03-interview-flight-data-tool/unsuitable-summary.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/03-interview-flight-data-tool/whole-tool-flights.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/03-interview-flight-data-tool/whole-tool-interviews.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/04-report-builder-tool/'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/04-report-builder-tool/appendix-tables.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/04-report-builder-tool/buttons.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/04-report-builder-tool/checkboxes-est.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/04-report-builder-tool/checkboxes-sen.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/04-report-builder-tool/done.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/04-report-builder-tool/goal-table.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/04-report-builder-tool/johnson-table.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/04-report-builder-tool/navbar-est.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/04-report-builder-tool/navbar-sen.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/04-report-builder-tool/nboot.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/04-report-builder-tool/trips-plot.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/04-report-builder-tool/whole-tool-est.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/04-report-builder-tool/whole-tool-sen.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/07-final-report-content/'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/07-final-report-content/new-KuskoHarvEst-project.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/07-final-report-content/new-directory.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/07-final-report-content/new-project-info.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/07-final-report-content/new-project.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/07-final-report-content/odd-openers-example.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/07-final-report-content/odd-openers-verbatim.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/07-final-report-content/raw-data-files-example.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/07-final-report-content/raw-data-files-folder.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/07-final-report-content/setnet-effort-by-opener.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/common/addins-menu-build-reports.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/common/addins-menu-get-help.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/common/addins-menu-interview-flight-data.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/common/addins-menu-meta-data.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/04-documentation/screenshots/common/addins-menu.png'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/05-example-data/BBH_2020_06_12_EXAMPLE_ONLY.csv'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/05-example-data/CBM_2020_06_12_EXAMPLE_ONLY.csv'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/05-example-data/Flight_counts_2020_06_12_EXAMPLE_ONLY.csv'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/07-final-report-content/00-specify-odd-openers.R'
Warning in utils::tar(filepath, pkgname, compression = compression, compression_level = 9L,  :
storing paths of more than 100 bytes is not portable:
'KuskoHarvEst/inst/rstudio/templates/project/resources/07-final-report-content/01-compile-all-estimates.R'
```
</details>

**Comment by bstaton1** [comment]
I've figured out how to tell the byte size of these paths:

```R
paths = readLines("C:/Users/bstaton/Desktop/bad-paths.txt")
paths = paths[stringr::str_detect(paths, "^'KuskoHarvEst")]
nchar(paths, type = "bytes")
```

They range from 101 to ~150. I think this will require more than a simple move to the `rstudio` or `templates` directories.

**Comment by bstaton1** [comment]
- [ ] Moving up two levels
- [ ] Renaming `screenshots` to be `SS`
- [ ] Renaming `04-documentation` to `04-docs`

Through those actions, I've reduced this list down to these items:

- [ ] `KuskoHarvEst/inst/rstudio/templates/04-docs/KuskoHarvEst-final-report/2-fig/KuskoHarvEst-Workflow.drawio`
- [ ] `KuskoHarvEst/inst/rstudio/templates/04-docs/KuskoHarvEst-final-report/2-fig/KuskoHarvEst-Workflow.pdf`
- [ ] `KuskoHarvEst/inst/rstudio/templates/04-docs/KuskoHarvEst-final-report/3-submit/KuskoHarvEst-final-report.pdf`
- [ ] `KuskoHarvEst/inst/rstudio/templates/04-docs/KuskoHarvEst-final-report/KuskoHarvEst-final-report.Rproj`
- [ ] `KuskoHarvEst/inst/rstudio/templates/04-docs/SS/03-interview-flight-data-tool/navbar-interview-options.png`
- [ ] `KuskoHarvEst/inst/rstudio/templates/04-docs/SS/03-interview-flight-data-tool/navbar-interview-output.png`
- [ ] `KuskoHarvEst/inst/rstudio/templates/04-docs/SS/03-interview-flight-data-tool/whole-tool-interviews.png`
- [ ] `KuskoHarvEst/inst/rstudio/templates/04-docs/Staton - 2018 - In-season harvest and effort estimates.pdf`

**Comment by bstaton1** [comment]
Closed by #224
